src/AppBundle/Repository/ProblemRepository.php

```diff
@@ -10,4 +12,17 @@
  */
 class ProblemRepository extends \Doctrine\ORM\EntityRepository
 {
+    public function getResolved(User $user) {
+        $qb = $this->createQueryBuilder('p');
+
+        $query = $qb
+            ->innerJoin('AppBundle:Solution', 's', 'WITH', 's.problem = p')
+            ->andWhere('s.user = :user')
+            ->setParameter('user', $user)
```

Comment:
You should add extra statement. Only accepted problems

Reply:
Done